base/uk.ac.stfc.isis.ibex.opis/check_opi_format_tests.py

```diff
@@ -474,6 +474,34 @@ def test_that_if_there_are_two_graphs_with_different_buffer_sizes_there_is_no_er
         # Assert
         assert_that(errors, has_length(0))
 
+    def test_that_if_a_graph_has_an_empty_trigger_pv_then_error(self):
+        # Arrange
+        root = etree.fromstring("""
```

Comment:
Can you use the `WIDGET_XML`  strings you created as part of the Eurotherm OPI background ticket (which has been merged into master and this branch) to tidy this up?